pkg/kubelet/dockertools/docker.go

```diff
@@ -316,7 +316,11 @@ func (d *dockerContainerCommandRunner) PortForward(podInfraContainerID string, p
 	}
 
 	containerPid := container.State.Pid
-	// TODO use exec.LookPath for socat / what if the host doesn't have it???
+	// TODO what if the host doesn't have it???
```

Comment:
the question about what to do is still valid, for this moment, it is only failing with an error.
If you think that this is really sufficient, I can remove it.